static/app/utils/profiling/colors/utils.tsx

```diff
@@ -0,0 +1,208 @@
+import {Frame} from '../frame';
+
+import {Color, FlamegraphTheme, LCH} from './../flamegraph/FlamegraphTheme';
+
+const uniqueBy = <T,>(arr: ReadonlyArray<T>, predicate: (t: T) => unknown): Array<T> => {
```

Comment:
@Zylphrex I think it's because I wanted to omit null and undefined keys, but I'm not sure. We also didn't use lodash at Specto, so this was a quick implementation. Tbh, this should be replaced by an implementation that can squeeze out all the performance possible. We often use uniqueBy on large arrays that can exceed 20k+ elements and I want to control the implementation 😅

Reply:
👍 We can revisit this in the future if necessary.